modin/core/io/text/text_file_dispatcher.py

```diff
@@ -1110,8 +1123,13 @@ def _read(cls, filepath_or_buffer, **kwargs):
             nrows=None,
             compression=compression_infered,
         )
+        fname = partition_kwargs.pop("fname")
+        kwargs_ref = cls.put(partition_kwargs)
         partition_ids, index_ids, dtypes_ids = cls._launch_tasks(
-            splits, **partition_kwargs
+            fname,
+            kwargs_ref,
+            splits=splits,
+            num_splits=num_splits,
```

Comment:
splits and num_splits look confusing. we could probably serialize kwargs inside of launch_tasks.

Reply:
`launch_tasks` is a common function for several dispatchers, so in order not to make changes to, for example, `read_custom_text` function, I do not serialize inside it.